app/models/cohort_columns/hmis_destination.rb

```diff
@@ -0,0 +1,12 @@
+module CohortColumns
+  class HmisDestination < ReadOnly
+    attribute :column, String, lazy: true, default: :hmis_destination
+    attribute :translation_key, String, lazy: true, default: 'Exit Destination (HMIS)'
+    attribute :title, String, lazy: true, default: -> (model, attr) { _(model.translation_key)}
+
+    def value cohort_client
+      cohort_client.client.last_exit_destination
```

Comment:
This will need to be pre-calculated since it doesn't live on client or cohort client.  See app/models/cohort_columns/household_members.rb for an example.  If it isn't pre-calculated it will significantly slow down cohort loading and cause N+M queries.